sample-app/build.gradle

```diff
@@ -39,6 +40,29 @@ android {
     lintOptions {
         abortOnError false
     }
+    
+    DomainObjectSet<BaseVariant> variants = getApplicationVariants() // or getLibraryVariants() in libraries
+    variants.addAll(getTestVariants())
+    variants.addAll(getUnitTestVariants())
+    variants.configureEach { variant ->
+        variant.getJavaCompileProvider().configure {
+            options.errorprone {
+                check("NullAway", CheckSeverity.ERROR)
+                option("NullAway:AnnotatedPackages", "com.uber")
+            }
+        }
+    }
```

Comment:
Have we tried manually injecting a `null` dereference on the sample app and seeing that NullAway does trigger on it? (e.g. uncomment https://github.com/uber/NullAway/blob/master/sample-app/src/main/java/com/uber/myapplication/MainActivity.java#L33 do `.\gradlew build`, observe error).

I am not sure we have integration tests for this config, so is worth doing that small manual sanity check :)